graviti/dataframe/indexing.py

```diff
@@ -5,59 +5,141 @@
 
 """The implementation of the Graviti indexing related class."""
 
-from typing import TYPE_CHECKING, Any, Generic, Iterable, TypeVar, Union, overload
+from typing import TYPE_CHECKING, Any, Iterable, TypeVar, Union, overload
+
+from graviti.utility import NestedDict
 
 if TYPE_CHECKING:
+    from graviti.dataframe.column.series import Series as ColumnSeries
     from graviti.dataframe.dataframe import DataFrame
     from graviti.dataframe.row.series import Series as RowSeries
-    from graviti.dataframe.series import SeriesBase
 
 _T = TypeVar("_T", int, str)
 
 
-class SeriesILocIndexer(Generic[_T]):
-    """Index class for Series.iloc."""
+class RowSeriesILocIndexer:
```

Comment:
since the index base class is removed, should we move the index classes into row and column folder?